xgi/classes/reportviews.py

```diff
@@ -413,25 +413,23 @@ def members(self, e=None, dtype=list):
             If `e` is a slice or if `e` does not exist in the hypergraph.
 
         """
-        if e is None:
-            if dtype is dict:
-                return {key: self._id_dict[key] for key in self._ids}
-            elif dtype is list:
-                return [self._id_dict[key] for key in self._ids]
-            else:
-                raise XGIError(f"Unrecognized dtype {dtype}")
-
-        if e not in self._ids:
-            raise IDNotFound(f"Item {e} not in this view")
-
         try:
             return self._id_dict[e].copy()
-        except TypeError as e:
+        except TypeError:
             if isinstance(e, slice):
                 raise XGIError(
                     f"{type(self).__name__} does not support slicing, "
                     f"try list(H.edges)[{e.start}:{e.stop}:{e.step}]"
```

Comment:
Do we realistically expect users to call `members` with a slice as an argument? I wouldn't expect this to be a frequent occurrence, so I vote we delete this check.

Reply:
Good call. Now everything (list, dict, slice, etc.) is caught as a TypeError, which I do think is important to handle.